test/migration/verify/verify_migration_test.go

```diff
@@ -119,6 +122,43 @@ func verifySecondMemberProvisionedSpace(t *testing.T, awaitilities wait.Awaitili
 	cleanup.AddCleanTasks(t, awaitilities.Host().Client, userSignupForSpace)
 }
 
+func verifyProvisionedSubSpace(t *testing.T, awaitilities wait.Awaitilities) {
+	parentSpace, _ := VerifyResourcesProvisionedForSpace(t, awaitilities, migration.ProvisionedParentSpace)
+	userSignupForSpace := checkMURMigratedAndGetSignup(t, awaitilities.Host(), migration.ProvisionedParentSpace)
+	targetClusterRoles := []string{cluster.RoleLabel(cluster.Tenant)}
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member2()
+	memberCluster, found, err := hostAwait.GetToolchainCluster(t, cluster.Member, memberAwait.Namespace, nil)
+	require.NoError(t, err)
+	require.True(t, found)
+
+	subSpace, err := awaitilities.Host().WaitForSubSpace(t,
+		migration.ProvisionedSpaceRequest,
+		memberAwait.Namespace,
+		parentSpace.GetName(),
+		wait.UntilSpaceHasTargetClusterRoles(targetClusterRoles),
+		wait.UntilSpaceHasTier("base"),
+		wait.UntilSpaceHasAnyProvisionedNamespaces())
+	require.NoError(t, err)
+
+	spaceRequest, err := memberAwait.WaitForSpaceRequest(t,
+		types.NamespacedName{
+			Namespace: memberAwait.Namespace,
```

Comment:
same here - the namespace come from the default namespace in parentSpace

Reply:
Fixed in 1048498.